### PR TITLE
Meta: Improve issue template chooser screen

### PR DESCRIPTION
**Pull request by fregante**
I think screen has too much text and an unexpected order. Unfortunately GitHub requires 2 lines so I added some emojis. Alternatives welcome

## Before

Screen Shot

## After

Screen Shot 3

### CODE REVIEW

**Comment by sindresorhus** [comment]
> Unfortunately GitHub requires 2 lines so I added some emojis. Alternatives welcome

How about zero width space?

**Comment by fregante** [comment]
I tried all of these spaces at once 3 times each: https://codepen.io/rubikzube/pen/wolgB

No luck. I suppose I could also use. `———`

**Comment by sindresorhus** [comment]
> I suppose I could also use. ——— (minimum 3 chars)

👍

File: .github/ISSUE_TEMPLATE/4_change_request.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-name: Change requests
+name: Complain
 about: Is there something you don't like? Let's talk
 labels: change request
```
Comment (fregante) [inline .github/ISSUE_TEMPLATE/4_change_request.md:4]: All good here? Feels like the verb is correct 😃

Comment (sindresorhus) [reply .github/ISSUE_TEMPLATE/4_change_request.md:4]: 👍

**Pull request merged**